### PR TITLE
Update for compatibility with Docker for Mac

### PR DESCRIPTION
**Pull request by nandofalcao**
Resolves the problem with permissions when generating a docker build via Jenkins

### CODE REVIEW

**Comment by nandofalcao** [comment]
https://github.com/jenkinsci/docker/issues/263

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,12 @@
 FROM jenkins
+
 USER root
+RUN apt-get -qq update \
+   && apt-get -qq -y install \
+   curl
+
+RUN curl -sSL https://get.docker.com/ | sh
```
Comment (wardviaene) [inline Dockerfile]: The reason why I do a manual download is to only keep the docker client, not the whole docker distribution. This saves some disk space. If you keep my original lines for those, I'll accept your PR

Comment (nandofalcao) [reply Dockerfile]: I changed it

**Review by nandofalcao — COMMENTED** [review]
The problem was in permissions. Tested in my environment (Mac). 👍

**Pull request closed without merging**